bot.py

```diff
@@ -467,6 +497,58 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def show_help(self, message, _command_content):
+        """
+        Show link to full documentation
+        """
+        p = self.COMMAND_PREFIX  # pylint: disable=invalid-name
+        reply = "```\n"
+
+        reply += f"{p}play <URL>          "
+        reply += "-> Play audio from URL\n"
+        reply += "\n"
+
+        reply += f"{p}play <playlist URL> "
+        reply += "-> Play all songs from playlist\n"
```

Comment:
This feature hasn't been implemented yet!

Reply:
Made playlist help comments